### PR TITLE
Use Github Action to get update infomation from macchrome/macstable automatically

### PR DESCRIPTION
**Pull request by S0urceC0der**
Use Github Action to get update information from `macchrome/macstable` and update repo  automatically.

PLEASE Add Github token to project secrets.

### CODE REVIEW

**Comment by mtslzr** [comment]
Awesome! Wanted to do this for a little while, but hadn't gotten to it. Will give it a once-over and get it in place. Thanks!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
       - name: Install Homebrew
         run: /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
       - name: Tap Repo
-        run: brew tap mtslzr/marmaduke-chromium
```
Comment (mtslzr) [inline .github/workflows/main.yml]: This should be swapped back before merging.

**Comment by mtslzr** [comment]
Added a TOKEN with push access.

Also, mind adding the new version number into the README as well, during the automatic commit? Otherwise, should be good to go.

**Comment by S0urceC0der** [comment]
Update the version number in README, but not sure the icon color has a specific meaning.

**Pull request merged**